staging/src/k8s.io/mount-utils/mount_helper_common.go

```diff
@@ -135,3 +143,14 @@ func removePathIfNotMountPoint(mountPath string, mounter Interface, extensiveMou
 	}
 	return notMnt, nil
 }
+
+// removePath attempts to remove the directory. Returns nil if the directory was removed or does not exist.
+func removePath(mountPath string) error {
+	klog.V(4).Infof("Warning: deleting path %q", mountPath)
+	err := os.Remove(mountPath)
+	if os.IsNotExist(err) {
```

Comment:
can you please update the comment when can errNotExist be returned?

Reply:
Recommendation now is to use: errors.Is(err, fs.ErrNotExist) https://pkg.go.dev/os#IsNotExist